apps/src/code-studio/components/progress/StudentTable.jsx

```diff
@@ -53,7 +53,21 @@ class StudentTable extends React.Component {
     students: PropTypes.arrayOf(studentShape).isRequired,
     onSelectUser: PropTypes.func.isRequired,
     getSelectedUserId: PropTypes.func.isRequired,
-    levels: PropTypes.array
+    levels: PropTypes.array,
+    sectionId: PropTypes.number,
+    scriptName: PropTypes.string
+  };
+
+  getRowLink = studentId => {
```

Comment:
there is some pretty complex logic here, so i think we can break this up to make it more readable. my initial thought would be something like:

```
let url;
const queryStr = `?section_id=${this.props.sectionId}&user_id=${studentId}`;

if (this.props.levels) {
  url = this.props.levels[0].bonus ? 'extras' : this.props.levels[0].levelNumber;
} else {
  url = this.props.scriptName;
}

return url + queryStr;
```